src/services/youtube.ts

```
@@ -52,4 +70,117 @@ export class YouTubeService extends BaseClient {
+   * Access playlist-related operations
+   */
+  get playlist(): YouTubeService.Playlist {
+    return this._playlist;
+  }
+}
+
+export namespace YouTubeService {
+  export class Channel extends BaseClient {
+    constructor(config: SupadataConfig) {
+      super(config);
+    }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)